src/nrepl/project-types.ts

```diff
@@ -100,7 +100,7 @@ async function selectShadowBuilds(connectSequence: ReplConnectSequence, foundBui
         placeHolder: "Select builds to start",
         saveAs: `${state.getProjectRootUri().toString()}/shadow-cljs-jack-in`
     }), aliases: string[] = menuSelections && menuSelections.cljAliases ? menuSelections.cljAliases.map(keywordize) : []; // TODO do the same as clj to prompt the user with a list of aliases
-    const aliasesOption = aliases.length > 0 ? `-A${aliases.join("")}` : '';
+    const aliasesOption = aliases.length > 0 ? `-M${aliases.join("")}` : '';
```

Comment:
I realized I still need to test this, and maybe add `: '-M'` here too.

Reply:
I don't get what cljAliases has to do with shadow builds... This function could use some tidying up. =)

Reply:
I think we should keep this PR change set small, as it is, and not mix in cleaning up of things, but if you really want me to, I'll do it.

Reply:
I removed the alias code from this function. Please review it.

Reply:
I guess now we must test this quite extensively. But I think it's good. If the aliases are not needed here, they really just create a smoke screen.

Reply:
Now tested and checked the original code. The clj aliases are actually needed. It picks up any `menuSelections:` -> `cljAliases:` from the custom connect sequence and will provide those to the shadow-cljs command as `-A` options, see: https://shadow-cljs.github.io/docs/UsersGuide.html#deps-edn

This is what it could look like in a connect sequence:

```json
        {
            "name": "bar",
            "projectType": "shadow-cljs",
            "cljsType": "shadow-cljs",
            "menuSelections": {
                "cljAliases": ["bar"],
                "cljsLaunchBuilds": ["calva-lib"]
            }
        }
```

Using this sequence will create this jack-in command line:

```sh
npx shadow-cljs -d cider/cider-nrepl:0.26.0 -A:bar watch calva-lib
```

Sorry for derailing this!

Reply:
Is there a way to trigger that jack-in command without using a connect sequence?

Reply:
I've reverted much of the changes and now there's only the change to use `-M` with the clj command line. I think this is all that's needed.

Reply:
Please merge if you feel it's good to go.